e2e/tsconfig.e2e.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../tsconfig.base.json",
```

Comment:
This file was deprecated in Angular 10.1. More details:

https://github.com/angular/angular-cli/pull/18478